qrenderdoc/Code/Interface/Extensions.h

```diff
@@ -908,7 +908,24 @@ passed nothing will happen.
 :param QWidget combo: The combo box.
 :param List[str] options: The new options for the combo box.
 )");
-  virtual void SetComboOptions(QWidget *combo, const rdcarray<rdcstr> &options) = 0;
+  virtual void SetComboBoxOptions(QWidget *combo, const rdcarray<rdcstr> &options) = 0;
```

Comment:
I don't generally promise compatibility on the python APIs, but I don't think breaking source compatibiltiy like this is justified for an arguably marginal at best improvement in consistency.

You could either leave the function as is and name the new functions `GetComboCount` and `SelectComboOption` or name the new functions as you have done and accept the minor difference.

Reply:
I will revert the name to the previous convention and change the naming of the new functions.